bin/roave-backward-compatibility-check.php

```diff
@@ -30,158 +30,106 @@
 use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Output\ConsoleOutput;
 use function file_exists;
+use function getcwd;
 
 (function () : void {
-    foreach ([__DIR__ . '/../vendor/autoload.php', __DIR__ . '/../autoload.php'] as $autoload) {
-        if (! file_exists($autoload)) {
-            continue;
+    (function () {
+        $autoloaderLocations = [
+            __DIR__ . '/../vendor/autoload.php',
+            __DIR__ . '/../autoload.php',
+            __DIR__ . '/../../autoload.php',
+            getcwd() . '/vendor/autoload.php',
+        ];
+
+        foreach ($autoloaderLocations as $autoload) {
+            if (file_exists($autoload)) {
+                /** @noinspection PhpIncludeInspection */
+                require_once $autoload;
+
+                return;
```

Comment:
There's an argument for not breaking when the first one is found, depending what use cases you want to support.

If I build a PHAR or do a `composer global require`, or any other method where the tool has its own deps locally then I need to include both the tool's autoloader _and_ the project's autoloader

Reply:
Wait this might not apply in your case if you're not depending on project reps being loaded

Reply:
> I need to include both the tool's autoloader and the project's autoloader

This can happen when using classes from this project directly, but not with the `bin/roave-backward-compatibility-check.php`, which is closed for customisation.

Reply:
Not sure what you mean...

I mean does this tool need to autoload target project's classes?

Reply:
No, it doesn't, but you can use classes from this tool in your own project

Reply:
OK then it's sufficient to find 'our' autoloader 👍

Reply:
Then :ship:

Lemme finish this :beer: and I'll release